### PR TITLE
New text snippets design 

### PR DESCRIPTION
**Pull request by dualcnhq**

### CODE REVIEW

File: src/tiles/text-snippet-block.ts [hunk 1]
```diff
@@ -90,32 +82,36 @@ export class TextSnippetBlock extends LitElement {
   static get styles(): CSSResultGroup {
     return css`
       .container {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        width: 100%;
+        border-left: 5px solid #194880;
```
Comment (latonv) [inline src/tiles/text-snippet-block.ts:89]: Can we add a 3px border-radius too? (I think this was accidentally omitted from the spec document, but it's present in the images).

**Review by latonv — APPROVED** [review]
LGTM, with two minor suggestions.

File: src/tiles/text-snippet-block.ts (hunk 1 above)
Comment (dualcnhq) [reply src/tiles/text-snippet-block.ts:89]: that's weird because in current offshoot prod, adding it back or removing it doesn't do anything 🤔

Comment (dualcnhq) [reply src/tiles/text-snippet-block.ts:89]: ohhhh... i notice it now, it's in the border-left corners

Comment (dualcnhq) [reply src/tiles/text-snippet-block.ts:89]: updated now

**Pull request merged**